src/molecules/BottomSheet/BottomSheet.native.js

```diff
@@ -191,9 +202,10 @@ const BottomSheet = ({
       panGestureComponentEnabled={true}
       adjustToContentHeight={adjustToContentHeight}
       alwaysOpen={initialHeight}
-      rootStyle={styles.rootStyle({ theme })}
+      rootStyle={styles.rootStyle({ theme, isSectionList })}
+      sectionListProps={isSectionList ? contentComponent[0].props.children.props : undefined}
```

Comment:
This is a major assumption that contentComponent's 1'st child will always be a SectionList. Wouldn't it be better for the API to have a `<BottomSheet.SectionList />` and we filter and find it and use that here?